src/imgtools/io/loaders.py

```
@@ -207,35 +232,52 @@
-                 csv_path_or_dataframe,
-                 colnames=[],
-                 seriesnames=[],
-                 id_column=None,
-                 expand_paths=False,
-                 readers=None):
 
+class ImageCSVLoader(BaseLoader):
+    def __init__(
+        self,
+        csv_path_or_dataframe,
+        colnames=[],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)